reject-escalated-privileges-yaml/reject-escalated-privileges-yaml.rb

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env ruby
+
+require "json"
+require "octokit"
+require "yaml"
+
+require File.join(File.dirname(__FILE__), "github")
+
+# can expand this list spliting with spaces. e.g %w(cluster-admin root webops)
+STRING_LIST = %w[cluster-admin]
+
+# Output the yaml file and the code if any of the strings in the STRING_LIST is
+# present in the files.
+
+# The below code recurse through each of hash and array and pattern match if
+# any of the string is present in any of the key or value field.
+
+def main(gh)
+  yaml_files_in_pr(gh).find_all { |file|
+    hash = YAML.load_file(file)
+    pattern = Regexp.union(STRING_LIST)
```

Comment:
This line is being executed for each yaml file in the PR, but it doesn't depend on the current yaml file, so I'd move it to above the loop, so that it only gets executed once.